samples/openapi3/client/petstore/java/jersey2-java8/src/main/java/org/openapitools/client/api/FakeApi.java

```diff
@@ -362,6 +362,66 @@ public String fakeOuterStringSerialize(String body) throws ApiException {
                                localVarHeaderParams, localVarCookieParams, localVarFormParams, localVarAccept, localVarContentType,
                                localVarAuthNames, localVarReturnType);
   }
+  /**
+   * Additional Properties with Array of Enums
+   * 
+   * @return Map&lt;String, List&lt;EnumClass&gt;&gt;
+   * @throws ApiException if fails to make API call
+   * @http.response.details
+     <table summary="Response Details" border="1">
+       <tr><td> Status Code </td><td> Description </td><td> Response Headers </td></tr>
+       <tr><td> 200 </td><td> Got object with additional properties with array of enums </td><td>  -  </td></tr>
+     </table>
+   */
+  public Map<String, List<EnumClass>> getAdditionalPropertiesWithArrayOfEnums() throws ApiException {
```

Comment:
It looks like Java is suffering the same issue with missing import and not using named class for mapping.

@bbdouglas (2017/07) @sreeshas (2017/08) @jfiala (2017/08) @lukoyanov (2017/09) @cbornet (2017/09) @jeff9finger (2018/01) @karismann (2019/03) @Zomzog (2019/04) @lwlee2608 (2019/10) @bkabrda (2020/01)